src/lib/shady-render.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {render as baseRender, Template, templateCaches, TemplateResult} from '../lit-html.js';
+
+export {html, svg, TemplateResult} from '../lit-html.js';
```

Comment:
Why export `html` and `svg` here?

Reply:
Just for convenience. This is everything that would be needed for the base class. I can leave it out.

Reply:
Gotcha... eh, I think it's ok. Wanted to make sure it was not a mistake.